doc/api/dns.md

```diff
@@ -174,14 +174,12 @@ Valid values for `rrtype` are:
  * `'NAPTR'` - name authority pointer record
 
 The `callback` function has arguments `(err, addresses)`. When successful,
-`addresses` will be an array, except when resolving an SOA record which returns
-an object structured in the same manner as one returned by the
-[`dns.resolveSoa()`][] method. The type of each item in `addresses` is
-determined by the record type, and described in the documentation for the
-corresponding lookup methods.
+`addresses` will be an array of results. A result will be a string except when
```

Comment:
IP addresses, hostnames (for CNAME) or text (for TXT). You think we should go into su much detail here?